sdk/python/kfp/compiler/compiler.py

```diff
@@ -140,23 +143,21 @@ def _op_to_template(self, op):
     if op.memory_request or op.cpu_request:
       template['container']['resources']['requests'] = {}
       if op.memory_request:
-        template['container']['resources']['requests']['memory'] = op.memory_request
+        template['container']['resources']['requests'][
```

Comment:
Minor issue: It would be best if these lines are cut in the more natural places, for example after ['memory']?

Reply:
ack. thanks